### ISSUE TITLE
Support SSL

### ISSUE THREAD

**Issue by cgay** [dylan-lang/http#7]
Integrate ssl-network into the server.

**Comment by PowellDean** [comment]
Hi:

I am trying to use the http library, as per the documentation. I am running Ubuntu 22.04 LTS and have the latest OpenDylan release, onto which I have installed the openssl-dev package. I have compiled a short program that uses the http library, but when I run it I receive the error:

```
Internal Error in socket-manager
```

The issue happens whether I get http:// or https://. Is my issue related to this? Or am I just not reading some of the documentation correctly?

Background:

In a clean new application folder, my package.dylan file looks like so:
```
{
    "dependencies": [ "http" ],
    "dev-dependencies": [ "testworks" ],
    "description": "YOUR DESCRIPTION HERE",
    "name": "hw2",
    "version": "0.1.0",
    "url": "https://github.com/#f/hw2",
    "keywords": [ ],
    "contact": "https://github.com/Unknown/hw2/issues",
    "license": "",
    "license-url": ""
}
```

src/app/library.dylan looks like this:

```
{
    "dependencies": [ "http" ],
    "dev-dependencies": [ "testworks" ],
    "description": "YOUR DESCRIPTION HERE",
    "name": "hw2",
    "version": "0.1.0",
    "url": "https://github.com/#f/hw2",
    "keywords": [ ],
    "contact": "https://github.com/Unknown/hw2/issues",
    "license": "",
    "license-url": ""
}
```

From the http document, I have the following line in main.dylan:
```
let response :: <http-response> = http-get("http://opendylan.org");
```

Thank you.

Dean Powell
Edmonton, Canada

**Comment by PowellDean** [comment]
@cgay Yes, the solution you suggested works just fine. Calling start-sockets() gave me the content from the successful http-get() call. You can close this issue.

Thank you. I apologize for the faulty copy/paste above in the original question.
